easybuild/easyconfigs/p/pkg-config/pkg-config-0.29.2-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'pkg-config'
```

Comment:
This EasyConfig can be removed if the build deps are replaced with `pkgconf-2.0.3`